src/librustc_driver/lib.rs

```diff
@@ -777,11 +778,17 @@ fn usage(verbose: bool, include_unstable_options: bool) {
     } else {
         "\n    --help -v           Print the full set of options rustc accepts"
     };
-    println!("{}\nAdditional help:
+    let at_path = if verbose {
+        "    @path               Read whitespace separated options from `path`\n"
```

Comment:
This reminds me, would you be up for adding a UI test for `rustc --help` and `rustc --help --verbose`? That way we could review updates to the test suite whenever that help output changes.